cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+comment: An unusual high count of signal jumps or falls can indicate ESI stability issues.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-1A"
+
+[Term]
+id: QC:40000
+name: MS1 signal fall (10x) count
+def: "The count of MS1 signal decline by a factor of ten or more (10x) between two subsequent scans" [PSI:QC]
+comment: An unusual high count of signal jumps or falls can indicate ESI stability issues.
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-1B"
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 1+ over 2+
+def: Ratio of 1+ peptide count  over 2+ peptide count  in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A" NARROW []
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 3+ over 2+
+def: Ratio of 3+ peptide count over 2+ peptide count in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B" NARROW []
+
+[Term]
+id: QC:40000
+name: Charged peptides ratio 4+ over 2+
+def: Ratio of 4+ peptide count over 2+ peptide count in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+synoym: "IS-3C" NARROW []
+
+[Term]
+id: QC:40000
+name: Mean charge in identified spectra
+def: Mean charge in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Median charge in identified spectra
+def: Median charge in identified spectra
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Charged spectra ratio +1 over +2
+def: Ratio of 1+ spectra count over 2+ spectra count in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3A"  RELATED []
+
+[Term]
+id: QC:40000
+name: Charged spectra ratio +3 over +2
+def: Ratio of 3+ peptide count over 2+ peptide count in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3B"  RELATED []
+
+[Term]
+id: QC:40000
+name: Charged spectra ratio +4 over +2
+def: Ratio of 4+ peptide count over 2+ peptide count in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+synonym: "IS-3C"  RELATED []
+
+[Term]
+id: QC:40000
+name: Mean charge in all MS2
+def: Mean charge in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Median charge in all MS2
+def: Median charge in all MS2
+is_a: QC:4000003 ! single value
+is_a: QC:4000010 ! ID free
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: QC:40000
+name: Number of missed cleavages
```

Comment:
sum of all missed cleavages across all identifications? Does this make any sense?

Reply:
indeed, we have the mc distribution metrics later which should be informative, probably a template remnant.